corgi/core/models.py

```diff
@@ -317,7 +326,9 @@ def save_product_taxonomy(self):
 
 
 class SoftwareBuildTag(Tag):
-    tagged_model = models.ForeignKey(SoftwareBuild, on_delete=models.CASCADE, related_name="tags")
+    tagged_model = models.ForeignKey(
+        SoftwareBuild, on_delete=models.CASCADE, related_name="tags", db_column="tagged_model_uuid"
```

Comment:
It's the to_field attribute which does that. The db_column attribute defines the name of the foreign key column name. Since I used SQL directly to create the foreign key, I think it's better to explicitly set the column name to useto match the SQL.